spec/api/query_spec.rb

```diff
@@ -1,60 +1,42 @@
 require 'spec_helper'
```

Comment:
Since the tests in this file are hitting the search end point should we just move these tests to the search_spec? It's unclear to me why these were ever separate.